x/cert/client/cli/test_helpers.go

```diff
@@ -19,3 +19,14 @@ func TxCreateServerExec(clientCtx client.Context, from fmt.Stringer, host string
 
 	return clitestutil.ExecTestCLICmd(clientCtx, cmdCreateServer(), args)
 }
+
+// TxCreateServerExec is used for testing create client certificate tx
```

Comment:
This comment says TxCreateServerExec but the function is named TxCreateClientExec